cli/cmd/util.go

```diff
@@ -34,6 +34,8 @@ const (
 
 const promptForChanges = "Do you want to automatically apply the suggested changes [y/n]? "
 
+const ERADefaultConfig = "era-config.json"
+
```

Comment:
```suggestion
const eraDefaultConfig = "era-config.json"
```
Filename does not need to be exported, otherwise looks good to me, thanks